includes/gateways/stripe/includes/admin/class-give-stripe-admin-settings.php

```diff
@@ -565,9 +565,28 @@ public function stripe_connect_field( $value, $option_value ) {
 				<td class="give-forminp give-forminp-api_key">
 					<?php
 					if ( give_stripe_is_connected() ) :
-						$stripe_user_id = give_get_option( 'give_stripe_user_id' );
+						$stripe_user_id      = give_get_option( 'give_stripe_user_id' );
+						$modal_title         = __( 'Connected! <strong>Important: Now please configure the stripe webhook to finalize the setup.</strong>', 'give' );
+						$modal_first_detail  = sprintf(
+							'%1$s %2$s',
+							__( 'In order for Stripe to function properly, you must configure your Stripe webhooks. You can visit your Stripe Account Dashboard to add a new webhook endpoint for the following URL:', 'give' ),
+							'<strong>https://give.test/?give-listener=stripe</strong>'
```

Comment:
@mehul0810 Can we make this like dynamic, so it will show webhook URL current site.

Reply:
@ravinderk Made this dynamic. I missed it to change as was checking the UI of the modal popup. Thanks for pointing.